cli/kubernetes/provider.go

```diff
@@ -102,7 +102,6 @@ func (provider *Provider) CreateMizuAggregatorPod(ctx context.Context, namespace
 			},
 			DNSPolicy:                     core.DNSClusterFirstWithHostNet,
 			TerminationGracePeriodSeconds: new(int64),
-			// Affinity: TODO: define node selector for all relevant nodes for this mizu instance
```

Comment:
why do you remove this comment? 
I think we do need it (maybe it is already done?)

Reply:
ok I see it is done